IssueTracker/DataAccess/Data/IData/IIssueData.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DataAccess.Models;
+
+namespace DataAccess.Data.IData
+{
+    public interface IIssueData
+    {
+        Task AddAsync(Issue entity);
+        Task<IEnumerable<Issue>> GetAllAsync();
+        Task<Issue?> GetByIdAsync(int id);
+        Task UpdateAsync(Issue entity);
+        Task DeleteAsync(int id);
+        Task NextPreview(int id, string method = "next");
```

Comment:
this method needs a better name

Reply:
Change to NextStatusOfIssueAsync and PreviewStatusOfIssueAsync

Reply:
rename Preview to Previous

Reply:
i change in PreviousStatusOfIssueAsync